pages/services/couchbase/1.0.0-6.0.0/security/index.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: layout.pug
+navigationTitle:
+excerpt: Using native transport encryption, authentication and authorization on Couchbase
+title: Security
+menuWeight: 50
+model: /services/couchbase/data.yml
+render: mustache
+---
+
+
+Refer to the [Couchbase documentation](https://developer.couchbase.com/documentation/server/current/security/security-x509certsintro.html).
+
+
+# DC/OS {{ model.techName }} Security
+
+The DC/OS {{ model.techName }} service supports {{ model.techShortName }}'s native transport encryption, authentication, and authorization mechanisms. The service provides automation and orchestration to simplify the use of these important features.
+
+<p class="message--note"><strong>NOTE: </strong>These security features are only available on DC/OS Enterprise 1.10 and later.</p> 
+
+
+## Transport Encryption
+
+With transport encryption enabled, DC/OS {{ model.techName }} will automatically deploy all nodes with the correct configuration to encrypt communication via SSL.
+
+<p class="message--note"><strong>NOTE: </strong> Couchbase Server currently supports TLS for client interactions and cross data center replication. There is no TLS for the node to node communication. See the Couchbase documentation for more details.</p> 
+
+
+The service uses the [DC/OS CA](/latest/security/ent/tls-ssl/) to generate the SSL artifacts that it uses to secure the service. Any client that trusts the DC/OS CA will consider the service's certificates valid.
+
+#include /services/include/security-configure-transport-encryption.tmpl
+
+Service account and service account secret plus enabling TLS can also be configured when launching the {{ model.serverName }} via the DC/OS console.
```

Comment:
Should this be "Service account and service account secret, plus enabling TLS, can be configured when launching..."?

Reply:
Changed.